examples/jsm/controls/OrbitControls.js

```diff
@@ -1188,6 +1188,7 @@ var OrbitControls = function ( object, domElement ) {
 	if ( scope.domElement.tabIndex === - 1 ) {
 
 		scope.domElement.tabIndex = 0;
+		scope.domElement.style.outline = 'none';
```

Comment:
Please also update the version in `examples/js`.

Reply:
Do you mean just like this https://github.com/mrdoob/three.js/commit/a814dc95f924aefd5f6982104c65f5355cc6998e ? Not sure what's automatically generated and what's not.

Reply:
You always edit `examples/js` and generate the `jsm` version via `node utils/modularize.js`.